src/ui/main.qml

```diff
@@ -26,12 +26,21 @@ Window {
     flags: Qt.platform.os === "ios" ? Qt.MaximizeUsingFullscreenGeometryHint : Qt.Window
 
     visible: true
-    width: fullscreenRequired() ? maximumWidth : 360
-    height: fullscreenRequired() ? maximumHeight : Theme.desktopAppHeight
-    maximumHeight: height
-    maximumWidth: width
-    minimumHeight: height
-    minimumWidth: width
+
+    function getWidth() {
+        return fullscreenRequired() ? maximumWidth : Theme.desktopAppWidth;
+    }
+    function getHeight() {
+        return fullscreenRequired() ? maximumHeight : Theme.desktopAppHeight;
+    }
+
+    width: getWidth()
+    height: getHeight()
+    maximumHeight: getHeight()
```

Comment:
This means that maximumHeight is computed recursively. Wondering if there is a better way.

Reply:
There is! Will push something much better up shortly